lib/Parser/Parser.cpp

```diff
@@ -211,18 +212,28 @@ Token Lexer::getNextToken(std::string &ErrStr) {
   if (*Begin == '(') {
     ++Begin;
     const char *NumBegin = Begin;
-    bool KnownBitsFlag = false, MoreKnownBitsFlag = false, NumSignBitsFlag = false;
+    bool KnownBitsFlag = false, MoreKnownBitsFlag = false, NumSignBitsFlag = false,
+    DemandedBitsFlag = false;
     unsigned SignBits = 0;
     while (*Begin == '0' || *Begin == '1' || *Begin == 'x') {
       ++Begin;
       KnownBitsFlag = true;
     }
+    bool NonNegative = false;
```

Comment:
I don't understand why you're adding stuff about NonNegative, isn't this PR just about demanded bits?